src/fhir.js

```diff
@@ -50,30 +50,31 @@ class FHIRWrapper {
 }
 
 class PatientSource {
-  constructor(filePathOrXML) {
+  constructor(filePathOrXML, shouldCheckProfile = false) {
     this._index = 0;
     this._bundles = [];
+    this._shouldCheckProfile = shouldCheckProfile;
     this._modelInfo = load(filePathOrXML);
   }
 
   // Convenience factory method for getting a FHIR 1.0.2 (DSTU2) Patient Source
-  static FHIRv102() {
-    return new PatientSource(FHIRv102XML);
+  static FHIRv102(shouldCheckProfile = false) {
```

Comment:
I think that these static functions are where the options could be really helpful.  After all,
```
cqlfhir.PatientSource.FHIRv102({ shouldCheckProfile: true })
```
is so much more self-descriptive than
```
cqlfhir.PatientSource.FHIRv102(true)
```

Reply:
Agreed!